mindmeld/components/question_answerer.py

```diff
@@ -765,15 +793,16 @@ def validate(self):
     class FilterClause(Clause):
         """This class models a knowledge base filter clause."""
 
-        def __init__(self, field, field_info=None, value=None, range_gt=None, range_gte=None,
-                     range_lt=None, range_lte=None):
+        def __init__(self, field, field_info=None, value=None, query_type='keyword', range_gt=None,
+                     range_gte=None, range_lt=None, range_lte=None):
             """Initialize a knowledge base filter clause. The filter type is determined by whether
             the range operators or value is passed in.
             """
 
             self.field = field
             self.field_info = field_info
             self.value = value
+            self.query_type=query_type
```

Comment:
Nit: fix lint errors like this `self.query_type=query_type` -> `self.query_type = query_type`